### ISSUE TITLE
Reopen #17: TypeError: Cannot read property 'startsWith' of null 

### ISSUE THREAD

**Issue by runartrollet** [yitzchak/linter-spell-latex#18]
I think the problem is not fixed yet. After upgrading to newest versions of both `linter-spell`(0.12.3) and `linter-spell-latex`(0.9.1), I still get the same error as reported.

**Comment by yitzchak** [comment]
Upon adding a word or doing a spell check?

**Comment by runartrollet** [comment]
On doing a spell-check, with the latex-syntax. Works well with any other syntax.

**Comment by yitzchak** [comment]
Do you have any words in the latex dictionary in the settings page?

**Comment by yitzchak** [comment]
Do you have a words in the latex dictionary in settings?

**Comment by oscarlevin** [comment]
I confirm this is still an issue, when checking a latex document.  I don't understand what you mean by the "latex dictionary in the settings page."  I see Plain Text Words, Git Commit Words, AsciiDoc Words, and Markdown Words.  Could this be the problem?

Incidentally, my initial bug report indicated I was able to fix the problem on one machine by adding a word with the cursor in a different place.  I think this was incorrect, as that machine is also getting the error.

**Comment by yitzchak** [comment]
Look in the settings for `linter-spell-latex`. It has it's own word list. Probably not the issue, but worth a try.

**Comment by oscarlevin** [comment]
I do not see the word list there either.  in the `linter-spell-latex` settings I see only a checkbox for "check settings."  On the other hand, in `linter-spell-html` I do see the word list box in the settings.

**Comment by yitzchak** [comment]
@oscarlevin Something wrong then. I would check for an update for `linter-spell-latex` or uninstall and then reinstall. You might also look in your config settings and manually delete any settings for `linter-spell-latex`.

**Comment by runartrollet** [comment]
@yitzchak @oscarlevin  Same thing happens here. No settings for `linter-spell-latex` other than a checkbox for `Check comments`. I had a look in the `config.cson` located in my `.atom`-folder (OSX), and it doesn't list anything for `linter-spell-latex` and it lists  a `""`-item and two `null`-items. I deleded the `null`-items, and the settings for `TeX, LaTeX and BibTeX word` showed up again. I did a reset of Atom and it works now. Thanks for clearing this up.
